lib/IRGen/GenDecl.cpp

```diff
@@ -4940,6 +4940,10 @@ llvm::GlobalValue *IRGenModule::defineTypeMetadata(
     if (isa<ClassDecl>(nominal)) {
       adjustmentIndex = MetadataAdjustmentIndex::Class;
     }
+
```

Comment:
"Magic" constant, can you add a comment what is going or use a `MetadataAdjustmentIndex::NoTypeLayoutString` (assuming that is what is going on here).